### DISCUSSION TITLE
New User Help

### DISCUSSION THREAD

**Discussion by SakuraDr3am** [lucidrains/big-sleep#119] [Q&A]
How do I install this to use through cmd or Windows Subsystem for Linux (WSL)? Sorry i'm new to this

**Comment by LtqxWYEG** [comment]
You could use the colab notebook if you just want to play around with it.

This is my notebook for which I made a pull request to include it in the readme. It has more features and less bugs than the official ones
https://colab.research.google.com/drive/1zVHK4t3nXQTsu5AskOOOf3Mc9TnhltUO?usp=sharing
